README.md

```diff
@@ -55,3 +61,58 @@ If your proposal was closed because a core contributor determined that it was
 not worth pursuing and you feel that it was wrongly closed, then feel free
 to join the developer IRC group and have a more in-depth discussion with 
 other core developers about the feature.
+
+## How core developers evaluate proposals
+
+The following is a list of considerations that core developers use when deciding
+to accept, close, or leave a proposal open. It is intended to be useful for core
+developers when considering proposals and for proposal-makers in drafting their
+proposals.
+
+#### 1. Does the proposal comply with the rules?
+Read the proposal and check to see that it complies with the above-stated rules.
+If it does not, close the proposal.
+
+#### 2. How much support is the proposal receiving?
+Evaluate the amount of support the proposal is receiving. This is an ongoing
+analysis. If a proposal receives little support at first, it may receive
+additional support later on.
+
+#### 3. Can this proposal be implemented with an addon?
```

Comment:
This has to be weighed together with how useful/frequently needed the proposal would be. If the majority of users would benefit from it, it doesn't necessarily make sense to force it to be an addon.
We can refer to https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html#the-problem-has-to-be-complex-or-frequent

In general, we can refer to the whole page as one of the guiding principles for reviewing proposals (just like we use it for reviewing PRs).

Reply:
Actually many of the points that come below are covered by https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html - I guess your PR served as inspiration for @reduz :)

We *might* want to reduce redundancy by referring to that docs page instead of restating everything, but at the same time it's likely good to have things written clearly in the readme.

Reply:
Yea, there is definitely a lot of overlap. But they are still addressing two distinct processes (PRs v. Proposals). My preference is to keep the two separate, as most of the proposals we receive are from people who do not contribute to the engine and that doc is likely to confuse them as it isn't clearly aimed at proposals.

Reply:
I have chosen to keep this section mostly as is. Changing to "less likely" instead of "unlikely". The purpose behind each section is to show an independent factor that must be weighed. All factors should be weighed against each other. IMO it is clearer to say that this factor lowers chances while other factors raise chances instead of mixing all the factors into each.